src/wireclient.ts

```diff
@@ -71,7 +71,9 @@ if (typeof ((global || window as any).ArrayBufferView) === 'function') {
   };
 } else {
   const arrayBufferViews = [
+    // eslint-disable-next-line no-undef
```

Comment:
I think these should be defined in the global eslint config here:

https://github.com/remotestorage/remotestorage.js/blob/feature/typescript_conversion/.eslintrc.js#L12-L19